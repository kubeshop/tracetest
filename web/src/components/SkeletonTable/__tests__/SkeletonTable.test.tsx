import {render} from '@testing-library/react';
import SkeletonTable from '../SkeletonTable';

test('SkeletonTable', () => {
  const result = render(
    <SkeletonTable loading={false}>
      <h2>Whatever</h2>
    </SkeletonTable>
  );
  expect(result.container).toMatchSnapshot();
});
