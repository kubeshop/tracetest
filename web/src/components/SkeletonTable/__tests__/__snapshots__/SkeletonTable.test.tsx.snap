// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkeletonTable 1`] = `
<div>
  <div>
    <h2>
      Whatever
    </h2>
  </div>
</div>
`;
